### PR TITLE
readme, folder structure, lst_supply brainstorming

### PR DESCRIPTION
**Pull request by mtitus6**
- Added a readme file.  
- Added a stating folder structure 
- Added column structure for _mint, _burn, _supply sql files for discussion.

### CODE REVIEW

**Comment by mtitus6** [comment]
@Valdorff  I went a little crazy but I would like for you to take a look.

I rebuilt  two metrics using reusable primitives. 

1. RPL Supply
2. Price-Peg Ratio

Rebuilt metrics are here --> https://dune.com/rp_community/rocketpoolv2

#1 is pretty straight forward, not a lot of complexity there.  I think we can pretty easily validate these numbers make sense.
#2 I completely rethought this one.  I normalized everything to hour, then was able to smooth things out by taking averages by day.  This one is harder to verify 1 to 1 due to different time blocks.

Let me know if you like the POC and want to continue forward with this structure.

File: README.md [hunk 1]
```diff
@@ -1 +1,51 @@
-# dune_rp_community
\ No newline at end of file
+# Rocketpool Community Dune SQL Queries Repository
```
Comment (acontry) [inline README.md]:
```suggestion
# Rocket Pool Community Dune SQL Queries Repository
```
`Rocketpool` -> `Rocket Pool` everywhere

File: README.md [hunk 2]
```diff
@@ -1 +1,51 @@
...
+
+This repository follows semantic versioning for releases. Any significant updates or changes to queries will be reflected in the version number according to the rules of semantic versioning.
+
+## License
+
+This repository is provided under the [MIT License](LICENSE), granting permissions for use, modification, and distribution of the content within, with proper attribution.
+
+## Contact
+
+If you have any questions, suggestions, or concerns regarding this repository, feel free to contact us by opening an issue on GitHub.
```
Comment (acontry) [inline README.md]: Maybe can reference discord in some way?

**Review by acontry — APPROVED** [review]
Overall structure looks great!

Wonder if we should add a SQL linter as a [pre-commit hook](https://pre-commit.com/hooks.html) like sqlfluff to keep syntax standard - something to follow up on.

File: README.md [hunk 3]
```diff
@@ -1 +1,51 @@
...
+
+This repository follows semantic versioning for releases. Any significant updates or changes to queries will be reflected in the version number according to the rules of semantic versioning.
+
+## License
+
+This repository is provided under the [MIT License](LICENSE), granting permissions for use, modification, and distribution of the content within, with proper attribution.
+
+## Contact
+
+If you have any questions, suggestions, or concerns regarding this repository, feel free to contact us by opening an issue on GitHub or by joining the conversation in the [Rocket Pool Discord](https://discord.gg/rocketpool).
```
Comment (mtitus6) [inline README.md:49]: @acontry thanks for the feedback!  How does this look?

Comment (acontry) [reply README.md:49]: Looks great 👍

**Pull request merged**